README.md

```diff
@@ -101,8 +101,8 @@ Read more about migrations here: https://www.prisma.io/docs/concepts/components/
 <details>
   <summary>Testing GraphQL</summary>
 
-1. Run `yarn dev` to start the local server
-2. Run `yarn generate` to generate the types locally
+1. Run `yarn generate` to generate the types locally
```

Comment:
Now we can generate the types before the server runs!

Reply:
awesome!!